File: PRSbackendSolution/PRSbackend/Models/PurchaseRequestLineItem.cs
﻿using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace PRSbackend.Models
{
    public class PurchaseRequestLineItem
    {
        [JsonIgnore]
        public int Id { get; set; }
        public int Quantity { get; set; }
        public int PurchaseRequestId { get; set; }
        public virtual PurchaseRequest PurchaseRequest { get; set; }
        public int ProductId { get; set; }
        public virtual Product Product { get; set; }
    }
}

File: PRSmaster/PRSweb-master/PRSweb-master/ng/src/app/models/PurchaseRequestLineItem.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var PurchaseRequestLineItem = /** @class */ (function () {
    function PurchaseRequestLineItem(Id, Quantity, ProductId, PurchaseRequestId) {
        this.Id = Id;
        this.Quantity = Quantity;
        this.ProductId = ProductId;
        this.PurchaseRequestId = PurchaseRequestId;
    }
    return PurchaseRequestLineItem;
}());
exports.PurchaseRequestLineItem = PurchaseRequestLineItem;
